### DISCUSSION TITLE
Saving Multiple Tabs — All Without Opening the Windows Save Dialogue for Each One Individually

### DISCUSSION THREAD

**Discussion by touwys** [gildas-lormeau/SingleFile#1107] [Q&A]
How can one save multiple tabs with SingleFile to a specified directory, without first having to pass through the Windows Save Dialogue for each one? 

Thank you.

**Comment by gildas-lormeau** [comment]
This is possible by disabling `Filename > open the "Save as" dialog to confirm the file name` in the options page of SingleFile or by using SingleFile Companion Lite, see https://github.com/gildas-lormeau/single-file-companion-lite.

**Comment by touwys** [comment]
Thank you!

**Reply by gildas-lormeau** [reply]
You're welcome :)

**Comment by touwys** [comment]
Installation of SingleFile Companion failed. Flagged this error:

**singlefile_companion_lite.exe - Entry Point Not Found**

_The procedure entry point GetHostNameW could not be located in the dynamic link library ws2_32.dll._

**Reply by gildas-lormeau** [reply]
I just did a test on Win 11/Chrome and it works fine on my end.

**Reply by touwys** [reply]
> I just did a test on Win 11/Chrome and it works fine on my end.

Thanks, and apology: I should have added to my post the Operating System is Windows 7 x64.

**Reply by gildas-lormeau** [reply]
I would recommend to use Single File Companion instead, see https://github.com/gildas-lormeau/single-file-companion
